src/content/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent.mdx

```diff
@@ -139,23 +123,23 @@ The preferred way to install the Linux agent is through the [package manager](/d
   />
 </TechTileGrid>
 
-To use these installation links, you must be logged to your New Relic account.
-
-If you don't have a New Relic account yet, or if you prefer to follow the procedure manually, [see our tutorial](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-agent-linux-using-package-manager). For advanced install needs, you can deploy the agent using our tarball files in [assisted](/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-assisted-install-infrastructure-agent-linux) or [manual](/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-manual-install-infrastructure-agent-linux) mode.
+If you don't have a New Relic account yet, or if you prefer to follow the procedure manually, [see our tutorial](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux/). 
 
 ### Windows Server and 10 [#Windows]
 
-To deploy the agent on a Windows Server host, [install it using our MSI installer](/docs/infrastructure/install-configure-manage-infrastructure/windows-installation/install-infrastructure-windows-server-using-msi-installer). For a guided procedure, click the button below and follow the step-by-step instructions:
+For a guided procedure to install the agent on Windows, click the button below and follow the step-by-step instructions:
```

Comment:
As with Linux, I don't see much value on keeping the tiles pointing to the Guided Install. I'd remove it and only mention that for manual install via .msi go to: https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/